src/components/lyne-link-list/styles/lyne-link-list.component.scss

```diff
@@ -15,6 +15,14 @@ lyne-title {
   @include listReset();
 }
 
+.link-list--horizontal {
```

Comment:
This is a challenging one. I would recommend to remove the media query here. For the current use case you are working on, this makes sense but for other occasions we eventually would like to have a different behaviour. I would define this as a general rule for link-list--horizontal and define the explicit logic for the bottom footer part in a separate component. What do you think?

Reply:
I removed the breakpoint and will implement the direction swap within the footer context if this will be possible.